c_cxx/MNIST/CMakeLists.txt

```
@@ -4,5 +4,9 @@
 cmake_minimum_required(VERSION 3.13)
 
 add_executable(mnist MNIST.cpp)
+target_link_options(mnist PRIVATE "/SUBSYSTEM:WINDOWS")
 
-target_link_options(mnist PRIVATE "/SUBSYSTEM:WINDOWS")
\ No newline at end of file
+copy_ort_dlls(mnist)
+
+add_custom_command(TARGET mnist POST_BUILD
+                   COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/MNIST/mnist.onnx ${CMAKE_BINARY_DIR}/MNIST)
```

could we use TARGET_FILE_DIR here?
```suggestion
                   COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/MNIST/mnist.onnx $<TARGET_FILE_DIR:mnist>)
```

---

I set it up to copy to the default working directory VS uses when running tests - which is `$(ProjectDir)` in the VS values and is different to TARGET_FILE_DIR.

Updated to override the default working directory in VS to OutDir which is the same as TARGET_FILE_DIR.